modules/getting-started/pages/index.adoc

```diff
@@ -8,7 +8,7 @@ endif::[]
 Welcome to the Getting Started section, where you can familiarize yourself with the most popular Anypoint Platform tools that support everything you do with APIs and integration applications. The documentation in this section helps you work toward building reusable code to manage integration of your digital assets. The key to integrating digital assets is the https://www.mulesoft.com/lp/whitepaper/api/api-led-connectivity[API-led approach].
 
 [TIP]
-If you have prior knowledge of APIs, you may want to know about xref:3.8@mule-runtime::index.adoc[Mule Runtime], MuleSoft's flagship integration platform. APIs can work with Mule applications in many ways. Try out xref:apikit::index.adoc[APIkit] or create a  https://anypoint.mulesoft.com/apiplatform/anypoint-platform/#/portals[public developer portal] for customized access of APIs and other assets in your organization.
+If you have prior knowledge of APIs, you may want to know about xref:mule-runtime::index.adoc[Mule Runtime], MuleSoft's flagship integration platform. APIs can work with Mule applications in many ways. Try out xref:apikit::index.adoc[APIkit] or create a  https://anypoint.mulesoft.com/apiplatform/anypoint-platform/#/portals[public developer portal] for customized access of APIs and other assets in your organization.
```

Comment:
'the' before the API kit link?

Reply:
It's the name, a proper noun (not common noun)--so sounds odd, but no article.